packages/devtools_app/lib/src/utils.dart

```diff
@@ -979,13 +983,23 @@ class DevToolsFile<T> {
   final T data;
 }
 
-/// Logging to debug console only in debug runs.
-void debugLogger(String message) {
+bool get isDebugging {
```

Comment:
remove this and use the 
assert(() {

  return true;
}());

pattern used by Flutter.

Reply:
Done.